docs/DeveloperGuide.md

```diff
@@ -395,69 +452,22 @@ Those without any stars are user stories that were considered but will not be im
 **Extensions**
 
 * 1a. The list is empty.
-
-  Use case ends.
-
-
-* 2a. The system detects an error in the entered data.
```

Comment:
Just wondering if there was a reason why this extension was deleted?

Reply:
The extension wasn't deleted! I believe because of the ordering/grouping, Git marked this red and deleted. 

This extension is still here in line 457

![image](https://github.com/user-attachments/assets/4605ca22-70b8-4594-9b74-8b28162e5399)